### DISCUSSION TITLE
got an error in Git

### DISCUSSION THREAD

**Discussion by IceCreamGamesAdel** [community/community#64208] [Programming Help] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Help i want to use git to work with unity in a team but when i want to "commit to main" i get the error "warning: in the working copy of  _*project name*_ , LF will be replaced by CRLF the next time Git touches it ...

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by WilliamHCarter** [comment, marked as answer]
Hi! So this isn't a huge error, it has to do with the way different computers handle the "Enter" key (new lines) in files. Its a pretty easy thing to fix so that you don't get these error messages:

**Quick Fix:**

- Open your command line in your Unity project folder.
- Type `git config core.autocrlf` true and hit Enter. This tells Git to automatically handle these "Enter" key differences for you.

**Consistent Fix for Everyone:**

- Create a file in your project named .gitattributes.
- Put the following lines in it:
```
* text=auto
*.cs text eol=lf
*.unity text eol=lf
*.prefab text eol=lf
```

- This file ensures that everyone in your team will have the same settings and won't see this error.
- Make sure everyone in your team uses the .gitattributes file you created.

Now, when you commit and work with others, you should no longer see that error.
Hope this helps!

_if this answer was satisfactory, feel free to 'mark as answered' in the feed to help other users with similar questions_

**Reply by loganrjudy** [reply]
This quick fix doesn't work for me. I'm still getting these errors.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @IceCreamGamesAdel ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
